config/initializers/hets_agent_initializer.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# A service class that defines and sends the Hets version requirement.
+class HetsAgentIninializer
+  VERSION_REQUIREMENT = '>= 0.100.0'
+  WORKER_QUEUE_NAME = "hets #{VERSION_REQUIREMENT}"
+  EXCHANGE_NAME = 'ex_hets_version_requirement'
+
+  attr_reader :connection
+
+  def initialize
+    @connection = Bunny.new
```

Comment:
You can re-use the existing ActiveJob connection with `Sneakers::CONFIG[:connection]`